typing/typetexp.mli

```diff
@@ -15,6 +15,7 @@
 
 (* Typechecking of type expressions for the core language *)
 
+open Asttypes
```

Comment:
Personal taste: I would rather avoid adding a global open and use `Asttypes.override_flag` below.